docs/recipes/github-actions.md

```diff
@@ -0,0 +1,92 @@
+# Using semantic-release with [GitHub Actions](https://help.github.com/en/categories/automating-your-workflow-with-github-actions)
+
+## Environment variables
+
+The [Authentication](../usage/ci-configuration.md#authentication) environment variables can be configured with [Secret variables](https://help.github.com/en/articles/virtual-environments-for-github-actions#creating-and-using-secrets-encrypted-variables).
+
+For this example, you'll need to setup a `NPM_TOKEN` to publish your package to NPM registry and a `GH_TOKEN` to generate a release at GitHub.
+
+If you want to push changes in your `package.json` to your `master` branch before generate release, what is recommended, you may need to add [`@semantic-release/git`](https://github.com/semantic-release/git). With this plugin your `GH_TOKEN` must have permission to push commits to `master` branch.
```

Comment:
We never said it's recommended to push changes to `package.json` back to `master` using the git plugin. Many folks do it, but many don't, it's not a recommendation. It's a good idea to document the special case that requires a custom `GH_TOKEN`, but that should be a separate section addressing this use case specifically, not the default setup instruction

Reply:
Good, thanks! I removed this section.